sentry-compose-helper/build.gradle.kts

```
@@ -20,6 +20,8 @@ kotlin {
 
                 compileOnly(compose.runtime)
                 compileOnly(compose.ui)
+
+                implementation("androidx.annotation:annotation:1.9.1")
```

```suggestion
                compileOnly("androidx.annotation:annotation:1.9.1")
```
should be that right?